CHANGES.rst

```diff
@@ -20,9 +20,11 @@ Unreleased
     stricter about header encodings than PEP 3333. (`#2766`_)
 -   Allow custom CLIs using ``FlaskGroup`` to set the debug flag without
     it always being overwritten based on environment variables. (`#2765`_)
+-   Fix issue with port set to 0 being overriden by the default value
 
 .. _#2766: https://github.com/pallets/flask/issues/2766
 .. _#2765: https://github.com/pallets/flask/pull/2765
+.. _#2926: https://github.com/pallets/flask/issues/2926
```

Comment:
This is the wrong link, this PR would be `.pull/2928`. And needs to referenced on the entry line above. That sentence should end with a period.

Reply:
Adjusted. Just out of curiosity, how do I know if PR or issue should be linked?